### DISCUSSION TITLE
Using session.get on a WriteableSession

### DISCUSSION THREAD

**Discussion by SelectSweet** [maxcountryman/axum-sessions#31] [Q&A]
Hi, i am Trying to get a key from a session to use in an SQL query then destroy the session after the SQL query is done, but the username variable is panicking due to a None Value, the problem function is listed below

```
pub async fn logout(mut session: WritableSession) {
    let connection = establish_connection().await;

    let username = session.get::<String>("Username").unwrap();

    let loggedin = sqlx::query!(
        "UPDATE public.Login SET loggedin = $2 WHERE username = $1",
        username,
        false,
    )
    .fetch_all(&connection)
    .await.unwrap();
    println!("Logging out");
    session.destroy();
    
}

```

**Comment by maxcountryman** [comment]
We'd need to see more of your code to help. I'll also point you to [tower-sessions](https://github.com/maxcountryman/tower-sessions), which is a redesign of this crate.
